modules/turnos/routes/turno.ts

```diff
@@ -94,20 +92,24 @@ router.patch('/turno/agenda/:idAgenda', async (req, res, next) => {
         let query;
         // seteamos el cupo en -1 cuando la agenda no tiene límite de cupos
         if ((agendaRes as any).cupo > -1) {
+            if (((agendaRes as any).cupo === 0)) {
```

Comment:
Esta linea puede  reducirse. Hay par de paréntesis de más. SI el valor de cupo se espera que sea igual o mayor que cero, se puede evaluar directamente el valor booleano del número (0 = true, !0= false)
```suggestion
            if (!(agendaRes as any).cupo) {
```

Reply:
Eliminados los paréntesis (Pulgar arriba). 

Con respecto al valor del cupo, hay que contemplar el caso cupo < 0 de manera distinta a cupo > 0 ya que el primero son las agendas que no tienen cupo limitado y el segundo aquellas que si lo tienen pero todavía hay lugar. El caso cupo === 0 sería uno muy particular cuando por mucha casualidad, conexión lenta, etc. se solapan dos daciones de turno (Usuarios distintos) del ultimo cupo disponible en una agenda.